src/components/SQFormDialog/SQFormDialog.tsx

```diff
@@ -96,6 +100,8 @@ function SQFormDialog<Values extends FormikValues>({
   helperText,
   helperTextType = 'error',
   throwAlertOnCancel = true,
+  validateOnBlur,
+  validateOnChange,
```

Comment:
Same here about setting to `true` by default.

Reply:
same as above